package.json

```diff
@@ -18,21 +18,21 @@
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.4",
     "envy": "^2.0.0",
-    "expo": "^41.0.0",
+    "expo": "^42.0.0",
     "expo-app-loading": "^1.0.3",
-    "expo-font": "~9.1.0",
-    "expo-status-bar": "^1.0.4",
+    "expo-font": "~9.2.1",
+    "expo-status-bar": "~1.0.4",
     "history": "^5.0.0",
     "immer": "^9.0.1",
     "normalizr": "^3.6.1",
     "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
```

Comment:
Any special reason that we pin these?  As I see it the package-lock will/can pin them for us on a per-project basis.

Reply:
Expo CLI did this, but Expo/React-Native expects a very specific version of React so this is a nice indication that you shouldn't mess with these.

Reply:
Ah, didn't realize!  Sounds good 👍